sharding/indexHashedNodesCoordinator.go

```diff
@@ -553,11 +557,13 @@ func (ihgs *indexHashedNodesCoordinator) EpochStartPrepare(metaHdr data.HeaderHa
 	err = ihgs.setNodesPerShards(resUpdateNodes.Eligible, resUpdateNodes.Waiting, leavingNodesMap, newEpoch)
 	if err != nil {
 		log.Error("set nodes per shard failed", "error", err.Error())
+		return
```

Comment:
maybe don't return but leave it to see the "damage" done (changed/not changed lists etc)?

Reply:
ok. removed return